src/utils/markdown.ts

```diff
@@ -0,0 +1,4 @@
+import markdownit from "markdown-it";
+export function renderMarkdown(markdown: string) {
+  return markdownit().renderInline(markdown);
```

Comment:
maybe let's add autolinker here to be safe? otherwise the new markdown-it default seems to pretty GFM compatible, which is what I assume users will expect

Reply:
ok how about this - it would complexity things just a wee bit, considering all the top level interfaces, but what if we allowed the user to pass plugins via a config?
I can do this as a follow up if you prefer, not a huge priority

Reply:
You mean [linkify](https://github.com/markdown-it/linkify-it)?